src/app/globals.css

```
@@ -165,3 +165,11 @@
   line-height: 140%;
   letter-spacing: -0.025em;
 }
+
+@utility bg-btn-gradient {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)